Gulpfile.mjs

```diff
@@ -294,23 +299,49 @@ if (process.env.CIRCLE_PR_NUMBER) {
 
 const babelVersion =
   require("./packages/babel-core/package.json").version + versionSuffix;
-function buildRollup(packages, targetBrowsers) {
+function buildRollup(packages, buildStandalone) {
   const sourcemap = process.env.NODE_ENV === "production";
```

Comment:
Not related to this PR:
Do we need to include source maps in the release?
Currently `NODE_ENV` is only used in `prepublish-build`, I don't know if this is expected.

Reply:
Good catch. I think the source maps should be included, especially for a browser package like `babel-standalone`.